cli/src/commands/services/status/mod.rs

```diff
@@ -131,6 +132,13 @@ async fn render_handlers_status(
             InvocationState::Pending,
         ));
 
+        // Scheduled
```

Comment:
I think you also need to update the headers in `render_services_status` because here we are adding a new column.